okta-idx-kotlin/src/main/java/com/okta/idx/kotlin/dto/IdxRemediation.kt

```diff
@@ -171,7 +171,7 @@ class IdxRemediation internal constructor(
         /**
          * Describes an individual field within a form, used to collect and submit information from the user to proceed through the authentication workflow.
          */
-        class Field internal constructor(
+        data class Field internal constructor(
```

Comment:
This allows users to construct any new Field object after getting a Field currently. So in essence, this constructor is public, but indirectly. This is a Kotlin design bug and it will be fixed upstream, but just wanted to note this

Reply:
reverting this. This is not suppose to make it in the PR. I was modifying this to see if i can use the copy constructor for tests.